packages/fxa-admin-panel/src/components/Permissions/index.test.tsx

```diff
@@ -4,29 +4,36 @@
 
 import React from 'react';
 import { render, RenderResult } from '@testing-library/react';
-import { IUserInfo } from '../../../interfaces';
+import { IClientConfig } from '../../../interfaces';
 import { Permissions } from './index';
+import { AdminPanelGroup, guard } from 'fxa-shared/gaurds';
+import { mockConfigBuilder } from '../../lib/config';
 
-const testUser: IUserInfo = {
-  email: 'test@mozilla.com',
-  group: 'vpn_fxa_admin_panel_prod',
-  permissions: {
-    p1: {
-      name: 'Feature 1',
-      enabled: true,
-    },
-    p2: {
-      name: 'Feature 2',
-      enabled: false,
+export const mockConfig: IClientConfig = mockConfigBuilder({
+  user: {
+    email: 'test@mozilla.com',
+    group: {
+      name: AdminPanelGroup.SupportAgentProd,
+      level: 1,
     },
   },
-};
+});
+
+jest.mock('../../hooks/UserContext.ts', () => ({
+  useUserContext: () => {
+    const ctx = {
+      user: mockConfig.user,
+      setUser: () => {},
+    };
+    return ctx;
+  },
+}));
 
-describe('rendered', () => {
+describe.only('Permissions', () => {
```

Comment:
Stray `.only`, we have lint rules setup in other repos so could probably add it here too.

Reply:
I'll add the linter rule for this. I hate it when these sneak by.

Reply:
I ended up filing a maintenance issue for this. Using the .eslintrc configs from other projects ended up conflicting with syntax in the .tsx files. It turns out none of the react projects have an .eslintrc at the moment. We should add one, but doing so will probably likely require updating the fxa eslint plugin.